ios/browser/api/debounce/debounce_service.mm

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/ios/browser/api/debounce/debounce_service+private.h"
+
+#include <MacTypes.h>
+#include <string>
```

Comment:
this doesn't appear to be used here

Reply:
nope I removed this. Sorry I sometimes template things out of existing code and remove what is not needed.